src/tests/catwalk_tests/test_evaluation.py

```diff
@@ -309,19 +317,19 @@ def test_evaluating_early_warning(db_engine_with_results_schema):
         if subset is None:
             where_hash = ""
         else:
-            populate_subset_data(db_engine_with_results_schema,
-                                 subset, list(range(num_entities)))
+            populate_subset_data(
+                db_engine_with_results_schema, subset, list(range(num_entities))
+            )
             SubsetFactory(subset_hash=filename_friendly_hash(subset))
             session.commit()
             where_hash = f"and subset_hash = '{filename_friendly_hash(subset)}'"
-
+        print(subset)
```

Comment:
Was this just being used for debugging? Should probably remove (or turn into a log statement if it's helpful to keep around)

Reply:
```suggestion
```